package.json

```diff
@@ -82,13 +83,15 @@
     "element-theme-default": "1.4.13",
     "enzyme": "3.3.0",
     "enzyme-adapter-react-16": "1.1.1",
-    "eslint": "4.18.2",
+    "eslint": "5.1.0",
```

Comment:
This could make some conflicts. I am upgrading eslint to v5 in my company and having some issues with plugins and adapters not migrated to that.